app/controllers/api/v1/authentication_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::AuthenticationController < ApiController
+  skip_before_action :authenticate_user!
+
+  def create
+    user = User.find_by(email: params[:email])
+    if user.valid_password? params[:password]
+      render json: { token: JsonWebToken.encode(sub: user.id) }
+    else
+      render json: { errors: ['Invalid email or password'] }
+    end
+  end
+end
```

Comment:
- [ ] I got this error while trying to test your endpoint, please fix this issue before you move forward. Because we can't perform the remaining endpoints while we have an issue in signing.

![endpoint](https://user-images.githubusercontent.com/55635977/139811087-d114d6f2-3575-419d-8b19-9f0dd0779afe.png)

- [ ] You are required to work on the `Registration endpoint - **POST:** `/register` or `/signup``, because a user is supposed to sign in after signup.

Reply:
@Emmyn5600 I think you got the error because you're sending Text instead of JSON and I think you should wrap the email inside double quote

Reply:
Let me do it, If it works I will remove that suggestion, in the meantime you can work on the register endpoint